### PR TITLE
feat: import-styles & add routes

### PR DESCRIPTION
**Pull request by vermasavita**
import all the screens & add react routes

### CODE REVIEW

**Review by NamrataRaikwar2002 — APPROVED** [review]
Great work, code looks clean, file structure  is clean.

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,24 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Routes, Route } from "react-router-dom";
+import { ProductListing } from './pages/Products/ProductListing';
+import { Cart } from './pages/Cart/Cart';
+import { Wishlist } from './pages/Wishlist/Wishlist';
+import { Login} from './pages/Authentication/Login';
+import { Signup} from './pages/Authentication/SignUp';
+import { Home } from "./pages/home/Home";
+// import logo from "./logo.png";
```
Comment (ravikumar0403) [inline src/App.js:9]: remove commented code if not required

File: src/App.js [hunk 2]
```diff
@@ -1,41 +1,24 @@
...
 
 function App() {
   return (
     <div className="App">
```
Comment (ravikumar0403) [inline src/App.js:13]: you can remove this classname as well if you are not using it for styling

File: src/pages/Authentication/SignUp.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import { Footer } from "../../components/footer/Footer";
+import { Navbar } from "../../components/navbar/Navbar";
+import "./authentication.css";
+import { Link } from 'react-router-dom';
+
+const Signup = () => {
+  return (
+    <div className="container">
+      <Navbar />
```
Comment (ravikumar0403) [inline src/pages/Authentication/SignUp.jsx:9]: instead of importing navbar in every component, you can have it in the App.js itself

**Review by ravikumar0403 — APPROVED** [review]
Code is clean and folder structure is also good.
I've left a few comments

File: src/App.js (hunk 1 above)
Comment (vermasavita) [reply src/App.js]: Removed

File: src/App.js (hunk 2 above)
Comment (vermasavita) [reply src/App.js]: Noted

**Pull request merged**